### ISSUE TITLE
validation AP fluctuation and validation loss curve increasing

### ISSUE THREAD

**Issue by hcmea** [Megvii-BaseDetection/YOLOX#1715]
Hello, I am training a custom dataset with YOLOX-nano model. I did not change the nano training parameters except the number of classes which is 1 for my dataset.  The Validation AP 0.50:0.95 looks like this:
![image](https://github.com/Megvii-BaseDetection/YOLOX/assets/98423541/3d9fa508-4374-4273-b7fa-7bc09d8cbb9a)

Is it normal to see these fluctuations? 

The other issue is calculating validation loss to see if it is overfitting. Is anyone try to log validation loss? I modified the code a bit to calculate validation loss. But somehow it looks weird. It fluctuates a lot  and it has an increasing trend. 
![image](https://github.com/Megvii-BaseDetection/YOLOX/assets/98423541/4aa9fec4-4dff-44e1-8179-82977123c736)

And this is total training loss by the way:
![image](https://github.com/Megvii-BaseDetection/YOLOX/assets/98423541/69140c8e-b7c5-4c86-87d2-4c745e3ac354)

What is going wrong? Did anyone have any idea to fix this?
Thanks.

**Comment by wang1233211** [comment]
Did you solve it?
